utils/linux-install-openresty.sh

```diff
@@ -26,7 +26,7 @@ sudo apt-get update
 
 if [ "$OPENRESTY_VERSION" == "source" ]; then
     cd ..
-    wget https://raw.githubusercontent.com/api7/apisix-build-tools/v2.7.0/build-apisix-base.sh
+    wget https://raw.githubusercontent.com/api7/apisix-build-tools/master/build-apisix-base.sh
```

Comment:
Is it possible to set branches by means of environment variable injection？

Reply:
It can be done, but why should it be? The purpose of submitting this PR is to use the latest build script directly.

Reply:
This is an util script, not just for ci

Reply:
So we use the master branch by default, but other scenarios can set this variable?

Reply:
Yeah, i think this is better